model/task/db.go

```diff
@@ -2962,3 +2962,53 @@ func getGenerateTasksEstimation(ctx context.Context, project, buildVariant, disp
 
 	return results, nil
 }
+
+type pendingGenerateTasksResults struct {
+	NumPendingGenerateTasks int `bson:"pending_generate_tasks"`
+}
+
+func GetPendingGenerateTasks(ctx context.Context) (int, error) {
+	match := bson.M{
+		GeneratedTasksKey: bson.M{
+			"$ne": true,
```

Comment:
Can we double check this will apply to tasks that don't have the field set at all?